pkg/sensors/exec/cgroups_test.go

```diff
@@ -225,3 +232,151 @@ func TestCgroupNoEvents(t *testing.T) {
 		}
 	}
 }
+
+// Ensure that we get cgroup_{mkdir|rmdir} events
+func TestCgroupEventMkdirRmdir(t *testing.T) {
```

Comment:
What is the difference between the test in `TestCgroupEventMkdirRmdir` and `TestCgroupNoEvents`? They seem to share a lot of initialization. Is there a way to have a common function that tests both?

Potentially using t.Run to create a new subtest?

If not, then maybe we can have a common function for their initialization?

Reply:
The difference is that  `TestCgroupEventMkdirRmdir`  has `val.TgCgrpLevel = 3` set which is in main tetragon is set when we complete cgroup discovery when probing tetragon cgroup migration.

So if that one is set we expect an event, if it is not set means no events.

Reply:
Ok I improved the test now and explained why how we set the cgroup tracking level, marking it resolved.